generators/import-jdl/index.js

```diff
@@ -220,6 +255,8 @@ module.exports = class extends ImporterGenerator {
                 } catch (error) {
                     this.error(`Error while generating entities from the parsed JDL\n${error}`);
                 }
+
+                statistics.sendCrashReport('source', 'stack', 'generatorVersion', 'yorc', 'jdl');
```

Comment:
I didnt find this method implementation?

Reply:
I removed the implementation and I should have removed this call too. I got some issues with asking the user if he agrees to send the report as user inputs are just promise. It just gets swallowed by the flow of other terminal output, I just need the find out how to make the input a blocking instruction.

Reply:
you need to use the done method for that. A phase will not continue unless the done is called